phpBB/styles/prosilver/template/attachment.html

```diff
@@ -29,6 +29,14 @@
 		</dl>
 		<!-- ENDIF -->
 
+		{% if _file.S_AUDIO_FILE %}
+		<dl class="file">
+			<dt><audio class="file-audio-controls" controls="controls" preload="metadata"><source src="{{ _file.U_DOWNLOAD_LINK }}" type="{{ _file.MIMETYPE }}"></audio></dt>
+			{% if _file.COMMENT %}<dd><em>{{ _file.COMMENT }}</em></dd>{% endif %}
```

Comment:
again please fix it like above give the dd a class and do not use the em tag but instead style it using the class.
```
		{% if _file.COMMENT %}
			<dd class="c-file-comment">{{ _file.COMMENT }}</dd>
		{% endif %}
```